src/builder/arg_group.rs

```diff
@@ -270,7 +270,7 @@ impl ArgGroup {
     ///
     /// assert!(group.is_multiple());
     /// ```
-    pub fn is_multiple(&mut self) -> bool {
+    pub fn is_multiple(&self) -> bool {
```

Comment:
This looks to b a breaking change as you can no longer capture the function with the old signature ([example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=f0687cbd33c14d081d58cacd81b763a1)).

Options:
- Create an issue and we can flag it for v5
- Put this change behind the `unstable-v5` feature flag